packages/grid/_modules_/grid/models/gridOptions.tsx

```diff
@@ -38,10 +38,9 @@ export interface GridOptions {
    */
   headerHeight: number;
   /**
-   * Set the height/width of the grid inner scrollbar.
-   * @default 15
+   * Override the height/width of the grid inner scrollbar.
    */
-  scrollbarSize: number;
+  scrollbarSize?: number;
```

Comment:
Well should we allow to override the scrollbar size? How about when you want to customize the scrollbar in the grid?

Reply:
I'm asking because it doesn't seems that the prop is enough to customize the scrollbar of the grid. I mean, for instance, how would a developer use https://www.npmjs.com/package/react-perfect-scrollbar?

Reply:
Not entirely sure, but you can put it around the grid. And if you change the size of the scrollbar, you would need to pass it to the grid